src/lib.rs

```diff
@@ -192,6 +251,8 @@ where
     timer: Instant,
 }
 
+/// Wrapper for Time.
+///
```

Comment:
Leaving an empty line leads to the first to be used in the list on which the entry appears, and the rest to be used in the page for the entry.